### PR TITLE
add requestAbsoluteZoom and show image in the tests

### PR DESCRIPTION
**Pull request by pxalcantara**
#### **Overview**

- This PR make some adjustments in siyi_sdk to make available the function `requestAbsoluteZoom()`.

#### **What was added/changed in this update**

- Make available requestAbsoluteZoom() function

#### **Depends On:**

- N/A

#### **Related Issues:**

- N/A

#### **Notes:**

- N/A

### CODE REVIEW

File: tests/test_zoom.py [hunk 1]
```diff
@@ -24,14 +25,18 @@ def test():
         print("No connection ")
         exit(1)
 
+    cam_str = cam.getCameraTypeString()
```
Comment (claudiaelizabete) [inline tests/test_zoom.py:28]:
```suggestion
    camera_type = cam.getCameraTypeString()    
```

**Review by claudiaelizabete — APPROVED** [review]

File: tests/test_zoom.py [hunk 2]
```diff
@@ -24,14 +25,18 @@ def test():
         print("No connection ")
         exit(1)
 
+    cam_str = cam.getCameraTypeString()    
+    rtsp = SIYIRTSP(rtsp_url="rtsp://192.168.144.25:8554/main.264",debug=False, cam_name=cam_str)
+    rtsp.setShowWindow(True)
+    
     val = cam.requestZoomIn()
-    sleep(1)
+    sleep(0.5)
```
Comment (victorlico) [inline tests/test_zoom.py]: I indicate to comment why change this values of sleep.

**Review by victorlico — APPROVED** [review]

File: tests/test_zoom.py (hunk 2 above)
Comment (pxalcantara) [reply tests/test_zoom.py]: Don't need to change, I'll put back the previous value.

File: tests/test_zoom.py (hunk 1 above)
Comment (pxalcantara) [reply tests/test_zoom.py:28]: done

**Pull request merged**